uPortal-spring/src/main/java/org/apereo/portal/spring/security/evaluator/PortalPermissionEvaluator.java

```diff
@@ -87,6 +91,13 @@ else if (permission instanceof String) {
                             + targetDomainObject.getClass());
         }
 
+        if (this.logger.isTraceEnabled()) {
+            logger.trace(
+                    String.format(
+                            "In hasPermission() - owner=[%s], activity=[%s], targetId=[%s] ",
+                            activity.getOwnerFname(), activity.getActivityFname(), targetId));
+        }
```

Comment:
Simply `logger.trace("In hasPermission() - owner=[{}], activity=[{}], targetId=[{}]", activity.getOwnerFname(), activity.getActivityFname(), targetId);`